haas/client/user.py

```diff
@@ -50,3 +50,22 @@ def remove(self, user, project):
         return self.check_response(
                 self.httpClient.request("POST", url, data=payload)
                 )
+
+    def set_admin(self, username, privilege):
```

Comment:
I'll update the string to be a boolean, should I also put in an issue for create() using the string still?